src/marketing/newsletter-queue.md

```diff
@@ -35,13 +35,26 @@ _Add to Newsletter Queue_
    {:.bs-callout-info}
    Do not remove the instructions, which are required by law in many jurisdictions.
 
+1. To apply custom styles for newsletter, add them into **Newsletter Styles** field.
+
 1. When complete, click <span class="btn">Save and Resume</span>.
 
     The newsletter appears in the queue waiting to be processed.
 
-    ![Newletter queue]({% link images/images/newsletter-queue-saved.png %}){: .zoom}
+    ![Newsletter queue]({% link images/images/newsletter-queue-saved.png %}){: .zoom}
     _Newsletter Queue_
 
 ## Check for Problems
 
 On the _Admin_ menu, go to **Reports** > _Marketing_ > **Newsletter Problem Reports**.
+
+### Button bar
+
+{: .format-table }
+|Button|Description|
+|--- |--- |
+|<span class="btn">Back</span>|Returns to the Newsletter Templates page without saving changes.|
+|<span class="btn">Reset</span>|Resets any unsaved changes in the queue information form to their previous values.|
+|<span class="btn">Preview Template</span>|Opens a preview page in a separate tab.|
+|<span class="btn">Save and Resume</span>|Saves all made changes. Puts the newsletter to queue.|
```

Comment:
Is there really no difference between these last two?

Reply:
Hello @dobooth , thank you for your question. I have double-checked this one, and if we are talking about sending the newsletter to queue from Marketing > Communications > Newsletter Template - there is really no any difference between the buttons. 
If the "Save and Resume" button was clicked from template for the newsletter that is to be sent - new entity is created in queue. The same behaviour is for "Save Newsletter" button.
But, if we are talking about editing an item in queue (Marketing - Communications - Newsletter queue)- there is the following logic:
If an item in queue has "Pause" status, then by clicking "Save and Resume" btn - status will be changed to sending. 
The main problem is that "Save and Resume" button exists in both cases. 
Thanks!

Reply:
Excellent. Thank you for checking.